argoverse/config/argoverse1.1.yml

```diff
@@ -0,0 +1,54 @@
+SensorDatasetConfig:
+  _target_: argoverse.sensor_dataset_config.SensorDatasetConfig
+  dataset_name: argoverse1.1
+  ring_cam_fps: 30
+  stereo_cam_fps: 5
+
+  # all ring cameras are `landscape` aspect ratio
+  sensors:
+    _target_: argoverse.sensor_dataset_config.SensorDimensions
+    
+    ring_front_center: 
+      _target_: argoverse.sensor_dataset_config.ImageDimensions
```

Comment:
Hmm, I would consider encapsulating the functionality of `ImageDimensions` into an entire `SensorConfig` (or something similar) with additional functionality.